### PR TITLE
feat: add new FAQ page

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Updates the FAQ page with the latest rewrite.

### CODE REVIEW

File: src/faqs/README.md [hunk 1]
```diff
@@ -2,50 +2,113 @@
...
+
+Users are required to wait for a period of one week when moving assets out of the Optimistic Ethereum mainnet. This period of time is called the Challenge Period and serves to help secure the assets stored on an Optimistic Ethereum network. You can find more information about the Challenge Period [here](https://community.optimism.io/docs/developers/bridging.html#understanding-the-fraud-proof-window). Please note that the [Optimistic Kovan testnet](https://community.optimism.io/docs/developers/networks.html#optimistic-kovan) has a Challenge Period of only 60 seconds to simplify the development process.
+
+### Are there transaction fees on Optimistic Ethereum?
+
+Yes, users are required to pay fees when sending transactions on an Optimistic Ethereum (just like on Ethereum). However, these fees are greatly reduced when compared to Ethereum. Refer to our [documentation page explaining transaction fees](https://community.optimism.io/docs/developers/fees.html) for more information. Alternatively, check out our [gas comparison page](https://optimism.io/gas-comparison) for concrete numbers comparing the cost of transactions on Ethereum to transactions on Optimistic Ethereum.
```
Comment (ben-chain) [inline src/faqs/README.md:50]: [documentation page explaining transaction fees](https://community.optimism.io/docs/developers/fees.html) should probably be updated with the medium doc here as we are placing more emphasis on it

File: src/faqs/README.md [hunk 2]
```diff
@@ -2,50 +2,113 @@
...
+### What infrastructure providers support Optimistic Ethereum?
+
+| Provider                                     | Service Type         | Mainnet Support | Kovan Testnet Support |
+| -------------------------------------------- | -------------------- | --------------- | --------------------- |
+| [Infura](https://infura.io/)                 | Node provider        | ☑               | ☑                     |
+| [Alchemy](https://www.alchemyapi.io/)        | Node provider        | ☑               |                       |
+| [QuickNode](https://www.quicknode.com/)      | Node provider        | ☑               | ☑                     |
+| [Etherscan](https://etherscan.io/)           | Block explorer       | ☑               | ☑                     |
+| [The Graph](https://thegraph.com/)           | Data indexing        | ☑               |                       |
+| [Dune Analytics](https://duneanalytics.com/) | Chain data analytics | ☑               |                       |
```
Comment (elenadimitrova) [inline src/faqs/README.md:73]: I added a comment to that in the notion document but basically we have a dedicated page for it in the community hub and might be not needed here.

Comment (smartcontracts) [reply src/faqs/README.md:73]: I assume that the page will be added in the new community hub refresh. If so, I will just leave it here for now and we can remove it as part of the refresh.

File: src/faqs/README.md [hunk 3]
```diff
@@ -2,50 +2,113 @@
...
+### What is Optimistic Ethereum?
+
+Optimistic Ethereum is an [EVM](https://ethereum.org/en/developers/docs/evm/)-compatible [Optimistic Rollup](https://research.paradigm.xyz/rollups) chain. It's designed to be fast, simple, and secure.
+
+<!--
+TODO: Need to wait until Dapp page is ready.
+
+## How can I use Optimistic Ethereum?
+
+Just check out one of the many dapps deployed to the Optimistic Ethereum mainnet and Optimistic Kovan testnet. If you're a developer, check out the [Networks and Connection Details page](https://community.optimism.io/docs/developers/networks.html).
```
Comment (qbzzt) [inline src/faqs/README.md:22]: 1. I wouldn't call two "many". I don't know how many we have on Optimistic Kovan, but unless we have links to them neither will the readers.
2. The DAPP app is going to be at `/docs/users/dapps.html`. It's nearly ready (just need Synthetix).

File: src/faqs/README.md [hunk 4]
```diff
@@ -2,50 +2,113 @@
...
+
+- [MetaMask](https://community.optimism.io/docs/developers/metamask.html)
+
+### How do I move assets into or out of Optimistic Ethereum?
+
+The easiest way to move assets into or out of an Optimistic Ethereum network is to use [Optimism's Gateway application](https://gateway.optimism.io/). We've written a [detailed walkthrough](https://community.optimism.io/docs/developers/eth-gateway.html) that explains how to use the Gateway.
+
+### Why do I need to wait a week when moving assets out of Optimistic Ethereum?
+
+Users are required to wait for a period of one week when moving assets out of the Optimistic Ethereum mainnet. This period of time is called the Challenge Period and serves to help secure the assets stored on an Optimistic Ethereum network. You can find more information about the Challenge Period [here](https://community.optimism.io/docs/developers/bridging.html#understanding-the-fraud-proof-window). Please note that the [Optimistic Kovan testnet](https://community.optimism.io/docs/developers/networks.html#optimistic-kovan) has a Challenge Period of only 60 seconds to simplify the development process.
```
Comment (qbzzt) [inline src/faqs/README.md:46]: It should be https://community.optimism.io/docs/developers/bridging.html#understanding-the-challenge-period

Comment (qbzzt) [reply src/faqs/README.md:46]: Also, we need to answer the question "Can I cancel my withdrawal during the challenge period" with a no.

File: src/faqs/README.md (hunk 1 above)
Comment (qbzzt) [reply src/faqs/README.md:50]: We moved that page to OPerations (/docs/operations/fees.html). I hope to finish the update on it today.

File: src/faqs/README.md [hunk 5]
```diff
@@ -2,50 +2,113 @@
...
+
+A Sequencer node is a special node in an Optimistic Ethereum network that can order transactions on short timescales (on the order of minutes). This opens up the door to very fast transaction confirmation times with strong guarantees about finality. Eventually, the operator of the Sequencer node on a network will be determined by some governing mechanism. However, for now, Optimism PBC operates the only such node on every major OE network.
+
+### Mainnet contract deployments are restricted by a whitelist
+
+Contract deployments to the Optimistic Ethereum mainnet are currently restricted by a whitelist. We are slowly introducing new applications to the mainnet system in order to reduce the available exploit surface and to throttle usage. Please fill out our [integration support signup form](https://docs.google.com/forms/u/1/d/e/1FAIpQLSdKyXpXY1C4caWD3baQBK1dPjEboOJ9dpj9flc-ursqq8KU0w/viewform) if you'd like to deploy an application to the OE mainnet. The [Optimistic Kovan testnet](https://community.optimism.io/docs/developers/networks.html#optimistic-kovan) is fully accessible to the general public.
+
+### Optimism's codebase is only partially audited
```
Comment (elenadimitrova) [inline src/faqs/README.md:97]: A lot of the commits below reference the archived repo. I cannot see anything referring to the monorepo.

Comment (smartcontracts) [reply src/faqs/README.md:97]: Hmmm would it be sufficient to explain that the old contracts repo has been archived and incorporated into the monorepo + a link to that monorepo?

File: src/faqs/README.md (hunk 3 above)
Comment (smartcontracts) [reply src/faqs/README.md:22]: Hmmm ok yeah I'm going to leave a TODO to fully rework this specific question + answer

File: src/faqs/README.md (hunk 1 above)
Comment (smartcontracts) [reply src/faqs/README.md:50]: Ok will redirect to medium page + leave a TODO to make sure we update this once the refresh is merged.

File: src/faqs/README.md (hunk 5 above)
Comment (smartcontracts) [reply src/faqs/README.md:104]: Added an asterisk with info

File: src/faqs/README.md (hunk 2 above)
Comment (smartcontracts) [reply src/faqs/README.md:80]: Added a TODO to remove this once the refresh is done

**Review by qbzzt — APPROVED** [review]

**Pull request merged**